js/modules.js

```diff
@@ -15,52 +15,79 @@ CORE.create_module("email-nav", function (sb) {
 CORE.create_module("email-catalog", function (sb) {
     var emails;
 
+    var openEmail = function(e) {
+        sb.hide();
+        sb.notify({
+            type : "open-email",
+            data : e.currentTarget.getAttribute("data-email-id")
```

Comment:
What happens if someone edits the DOM and changes the ID to something else? Does the original or the user changed value come here?

I got some questions about our IDs, is it a random guid or an incrementing value? Having an incrementing value here could introduce a security risk if our show email page isn't secure enough to prevent access to emails you don't own.